msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1528,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+### Receive meeting participant events
+
+Your bot can receive real-time meeting events such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events in Developer Portal.
+
+> [!NOTE]
+>
+> * Participant events are supported only for scheduled meetings.
+> * For a bot to receive participant events, ensure that you add the bot to the meeting before a participant joins or leaves the meeting.
+
+To subscribe to participant events, follow these steps:
+
+1. In [Developer Portal](https://dev.teams.microsoft.com/) open your bot app or import an existing app.
+1. In the **Meeting event subscriptions** section, select the events:
+    * Participant join
+    * Participant leave
+1. Select **Save**
+
+   :::image type="content" source="~/assets/images/apps-in-meetings/participant-events.png" alt-text="Screenshot shows how developer portal display for participant events.":::
+1. Ensure that the `OnlineMeetingParticipant.Read.Chat` RSC permission is configured in your app manifest.
+
+   If your app doesn't have the RSC permission, add it through the **Configure** > **Permissions** section of your app in Developer Portal. For more information, see [RSC permissions.](~/graph-api/rsc/resource-specific-consent.md)
+
+Following are the examples of the participant join and leave event payloads:
+
+# [Participant join event](#tab/participant-join-event) 
+
+The following code provides an example of the participant join event payload:
+
+```json
+{ 
+
+    "type": "event", 
+    "name": "application/vnd.microsoft.meetingParticipantJoin",
+    "timestamp": "2023-02-23T19:34:07.478Z", 
+    "channelId": "msteams", 
+    "serviceUrl": "https://smba.trafficmanager.net/amer/", 
+    "from": { 
+        "id": "29:id_xyz" 
+    }, 
+    "conversation": { 
+        "isGroup": true, 
+        "conversationType": "groupchat", 
+        "id": "19:meeting_threadId@thread.v2" 
+    }, 
+"recipient": { 
+    "id": "28:botid" 
+    },  
+    "value": { 
+       "members": [ 
+       { 
+        "user": { 
+            "tenantId": "tenantid", 
+            "objectId": "user_object_Id", 
+            "id": "29:userId ", 
+            "name": "Test User", 
+            "aadObjectId": " user_object_Id " 
+        },   
+        "meeting": { 
+            "inMeeting": true, 
+            "role": "Organizer" //Atendee, Organizer, Presenter 
```

Comment:
There is a typo in the comment of role property. It should be Attendee instead of Atendee

Reply:
Updated